src/main/java/de/codingair/tradesystem/utils/database/migrations/mysql/CreateTradeLogTableMigration.java

```diff
@@ -0,0 +1,20 @@
+package de.codingair.tradesystem.utils.database.migrations.mysql;
+
+import de.codingair.tradesystem.utils.database.migrations.Migration;
+
+public class CreateTradeLogTableMigration implements Migration {
+    @Override
+    public String getStatement() {
+        return "CREATE TABLE IF NOT EXISTS tradelog (\n"
+                + "	id BIGINT PRIMARY KEY AUTO_INCREMENT,\n"
+                + "	player1 varchar(255) NOT NULL,\n"
+                + "	player2 varchar(255) NOT NULL,\n"
+                + "	message text NOT NULL,\n"
+                + "	timestamp BIGINT NOT NULL" + ");";
```

Comment:
Hey man,
Thanks for the good review.
I removed the \n.
Changed the varchar, I didn't know how long player names could get and forgot to change this. Thanks for noticing.
I prefer a unix timesstamp since it's more universal overall it can be translated to any timezone afterwards.